webpack-common.js

```diff
@@ -118,13 +122,20 @@ export function getRules({ babelOptions, bundleStylesWithJs = false } = {}) {
     ];
   }
 
-  return [
-    {
+  let babelLoader = {};
+  // The main purpose for this is when we call getRules from the storybook setup
+  // (stories), it causes flow not to work.
+  if (includeBabelLoader) {
```

Comment:
Do you know why this isn't working with Flow?

Reply:
I dug in a little bit more and saw if I added the following as plugin to the babelrc file it would keep flow types here

"babel-plugin-react-docgen"

So I will make sure to include this : )